src/data/valid/Biosample-exhasutive.yaml

```diff
@@ -25,8 +25,8 @@ embargoed: true
 collected_from: nmdc:unconstrained_site_identifier_string
 type: nmdc:Biosample. change this to require a class name or an enumeration
 img_identifiers:
-  - any string 1
-  - any string 2
+  - prefix:any string 1
```

Comment:
Let's use a prefix that's defined in the schema. I think there are some cases in which using an undefined prefix causes problems. Maybe converting data to RDF TTL?